packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -7673,6 +7697,9 @@ export default class Meeting extends StatelessWebexPlugin {
       if (roles.includes(SELF_ROLES.COHOST)) {
         return 'cohost';
       }
+      if (roles.includes(SELF_ROLES.PRESENTER)) {
```

Comment:
Looks like the sequence of roles matters here.

Reply:
I think so and I think the Presenter takes precedence than attendee. Attendees would not have meeting control at all.